src/components/button/button_group/_button_group_button.scss

```diff
@@ -123,20 +123,43 @@
 .ouiButtonGroup--small {
   .ouiButtonGroupButton {
     border: $ouiBorderThin;
+  }
```

Comment:
```suggestion
    &:not(:first-child) {
      margin-left: -1px;
    }
  }
```

Reply:
This is set on line 133 under `&:not(.ouiButtonGroup--vertical)`

Reply:
I should not review PR's when im sleepy. yes you are right. I did not notice that the margin properties changed are different even though the values are the same.